akka-stream-tests/src/test/java/akka/stream/javadsl/BidiFlowTest.java

```diff
@@ -272,4 +272,10 @@ public Integer apply(String arg) {
     Arrays.sort(rr);
     assertArrayEquals(new Long[] { 3L, 12L }, rr);
   }
+  
+  public void mustSuitablyOverrideAttributeHandlingMethods() {
+    @SuppressWarnings("unused")
+    final BidiFlow<Integer, Long, ByteString, String, BoxedUnit> b =
+        bidi.withAttributes(Attributes.name("")).addAttributes(Attributes.asyncBoundary()).named("");
```

Comment:
Without the overrides the returned type would be a `Graph[BidiShape[...], ...]` instead of a BidiFlow.